### ISSUE TITLE
Wrong item rendering of virtualScroll which is inside ion-card.

### ISSUE THREAD

**Issue by Mohsen7s** [ionic-team/ionic#10317]
**Your Notes**
It' took me weeks to compare hybrid frameworks and choose Ionic for our company, Of course minus the time spent on learning and setup it!! It's really shameful to announce Ionic 2 as stable version with this horrible bug!

**Ionic version:** 
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**
[x ] bug report
[ ] feature request
[ ] support request 

**Current behavior:**
Using virtualScroll inside ion-card cause virtualScroll to render wrong items while scrolling.
Some of items of virtualScroll visible area aren't rendered and also some of them which are not in visible area are rendered.

**Expected behavior:**
1.Only those items of virtualScroll must be rendered that are in visible range.
2.Any items of virtualScroll outside visible range must not be rendered. [Usually first element of array is always rendered]

**Steps to reproduce:**
Simply put virtualScroll inside a ion-card.
Here is Plunker for lazy people.
http://plnkr.co/edit/1taMRHiGUVL1rRpoWrbN?p=preview

**Other information:**
Nothing more

**Ionic info:**
Your system information:
Cordova CLI: 6.5.0
Ionic Framework Version: 2.0.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 0.0.44
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.9.2
Xcode version: Not installed

**Comment by Mohsen7s** [comment]
Almost one month and not even a response...

**Comment by Mohsen7s** [comment]
@jgw96  @mlynch @manucorporat Shouldn't this issue have virtualscroll label and maybe 3.0 blocker too!?

**Comment by Mohsen7s** [comment]
Just upgraded to v3 and the problem still exist.. Does anybody care about virtual-scroll issues !?

Steps to reproduce for v3:
http://plnkr.co/edit/F3hFNnXuVOXpulBcUJv5?p=preview

**Comment by JustasKuizinas** [comment]
Virtual scroll is broken and not ready for production, try to find angular2 virtual list alternative
